2-mandatory/1-create-functions.js

```diff
@@ -3,15 +3,17 @@ Write a function that:
 - Accepts an array as a parameter.
 - Returns a new array containing the first five elements of the passed array.
 */
-function first5() {
+function first5(arr) {
+  return arr.filter((element, index) => index < 5);
```

Comment:
This is not a good approach, filtering is an expensive operation that iterates over each element in the array. If array has `n` elements, it will result in `n` operations. 

You can get the first 5 elements with 1 operation - `arr.slice(0, 5);`